src/elements/Branching.cpp

```diff
@@ -434,10 +438,10 @@ bool Branching::uniform_new_branch(TNodePtr &branching_node, NodePtr &new_node,
 
         for (auto &node : neurite_->nodes_)
         {
-            if (node.second->get_branch()->size() > 3)
+            if (node.second->get_branch()->get_length() > 2*latbranch_dist_)
             {
                 double key = powf(uniform_(*(rnd_engine).get()),
-                                  1. / node.second->get_branch()->size());
+                                  1. / node.second->get_branch()->get_length());
                 if (key > max)
```

Comment:
instead of using a number of segments, I use a distance from the root and the tip of the branch.
I will add a comment and make it a user-defined parameter, I guess... just need to find a name

Reply:
OK, good luck ;)

I tried with
no_branch_dist
no_branch_interval
branch_free_//
no_latbranc_dist

bla bla. None convincing